pkg/cli/admin/release/extract.go

```diff
@@ -349,20 +354,42 @@ func (o *ExtractOptions) Run(ctx context.Context) error {
 		}
 	}
 
-	tarEntryCallbacks := []extract.TarEntryFunc{}
+	var versionInImageConfig string
+	opts.ImageConfigCallback = func(imageConfig *dockerv1client.DockerImageConfig) {
+		if imageConfig == nil {
+			klog.Info("Cannot retrieve the version because no image configuration is provided in the image to extract")
+			return
+		}
+		if c := imageConfig.Config; c != nil {
+			if v, ok := c.Labels["io.openshift.release"]; ok {
+				klog.V(2).Infof("Retrieved the version from image configuration in the image to extract: %s", v)
+				versionInImageConfig = v
+			} else {
+				klog.Info("Cannot retrieve the version from image configuration in the image to extract")
```

Comment:
"Cannot" sounds bad so I'd expect this to be an error or warning but it is an info. Is that intentional?

Reply:
It was intensional as `versionInImageConfig` is used only for the logging and I was not sure if it should be there for each release.
Now it is changed to the error level (and let us see how if anyone is going to complain of it).

This part of the code is moved to https://github.com/openshift/oc/pull/2050/files